packages/core/src/components/Tooltip/index.tsx

```diff
@@ -18,6 +18,8 @@ const EMPTY_TARGET_RECT: ClientRect = {
 };
 
 export type TooltipProps = {
+  /** Accessibility label. If not specified, all tooltip content is duplicated in an accessibility portal rendered at all times. */
```

Comment:
small nit: to change the last part of the sentence to `is duplicated, rendered in an off-screen element with a separate layer.`